### PR TITLE
FastAI Recommender

### PR DESCRIPTION
**Pull request by danielsc**
### Description
Adding recommender based on fast.ai library

### Motivation and Context
Fast AI is an interesting library that uses Pytorch under the cover. This is an interesting addition to the current lineup of algorithms in this repo.

### Checklist:

- [x] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING).
- [x] I have added tests.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: notebooks/00_quick_start/fastai_recommendation.ipynb [hunk 1]
```diff
@@ -0,0 +1,788 @@
...
+      ]
+     },
+     "execution_count": 52,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Mean Squared Error\n",
+    "from sklearn.metrics import r2_score, explained_variance_score\n",
```
Comment (yueguoguo) [inline notebooks/00_quick_start/fastai_recommendation.ipynb]: We have rating metrics available in the reco_util functions which can be used directly. See [here](https://github.com/Microsoft/Recommenders/blob/master/reco_utils/evaluation/python_evaluation.py) for details.

File: notebooks/00_quick_start/fastai_recommendation.ipynb [hunk 2]
```diff
@@ -0,0 +1,788 @@
...
+       ")"
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "learn = collab_learner(data, n_factors=40, y_range=[0,5.5], wd=1e-1)\n",
```
Comment (miguelgfierro) [inline notebooks/00_quick_start/fastai_recommendation.ipynb]: I think the range is 1-5 for movielens 100k

File: notebooks/00_quick_start/fastai_recommendation.ipynb (hunk 1 above)
Comment (miguelgfierro) [reply notebooks/00_quick_start/fastai_recommendation.ipynb]: yeah +1 to this

Comment (danielsc) [reply notebooks/00_quick_start/fastai_recommendation.ipynb]: ok -- did that

File: notebooks/00_quick_start/fastai_recommendation.ipynb (hunk 2 above)
Comment (danielsc) [reply notebooks/00_quick_start/fastai_recommendation.ipynb]: Although ratings can only range from 1-5, I am are setting the range of possible ratings from 0 to 5.5 -- that will allow the model to predict values to both sides of 1 and 5, which improves accuracy.
I added that as an explanation to the cell above
